Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,81 @@
+/*Global Styles*/
+
+body{
+	background-color:#F6F6F6;
+	padding: 75;
+	border-bottom: 5px solid #000;
+	border-top: 5px solid #000;
+	font-family: 'Lora', serif;
+}
+
+h1{
```

Comment:
Here's where you could do a little consolidating:
```
h1, h2, h3 {
  font-family: 'Montserrat', sans-serif;
}
```
That way you're at least not repeating that many times over